third_party/istio-1.2.4/values.yaml

```diff
@@ -40,7 +40,10 @@ gateways:
     replicaCount: 2
     autoscaleMin: 2
     autoscaleMax: 4
-    resources: {}
+    resources:
+      requests:
+        cpu: 250m
+        memory: 256Mi
```

Comment:
Has this change been made intentionally?

Reply:
Yes, because the default is 10m which is abysmally low.